plugins/qcheck-stm/src/reserr.ml

```diff
@@ -130,9 +130,13 @@ let pp_kind ppf kind =
       pf ppf "Skipping unsupported modifies clause:@ %a" text
         "expected \"modifies x\" or \"modifies x.model\" where x is the SUT"
   | Ensures_not_found_for_next_state (f, m) ->
-      pf ppf "Skipping %s:@ model@ %s@ %a" f m text
-        "is declared as modified by the function but no translatable ensures \
-         clause was found"
+      pf ppf "@[Skipping %s:@ model@ %s@ %a@]@ @[%a@]@ @[%s = f (old %s)@]" f m
```

Comment:
I would suggest something like:
```ocaml
"Skipping %s:@ model@ %s %a@;%a@ x.%s = ..."
```
to make sure it’s clear that the `t` must appear, not `contents` by itself, and that the clause doesn’t _need_ to use `old`.